welcome to exception in python
